api/features/versioning/views.py

```diff
@@ -46,7 +47,6 @@
 )
 class EnvironmentFeatureVersionViewSet(
     GenericViewSet,
-    RetrieveModelMixin,
```

Comment:
As mentioned in the description, this is technically a breaking API change but, since the versioning feature has not been released to anyone yet, I don't see this as an issue.